rskj-core/src/main/java/co/rsk/peg/abi/ABICallElection.java

```diff
@@ -16,9 +16,10 @@
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
-package co.rsk.peg;
+package co.rsk.peg.abi;
 
 import co.rsk.core.RskAddress;
+import co.rsk.peg.AddressBasedAuthorizer;
```

Comment:
AddressBasedAuthorizer will also be moved to abi package right? Is that part of a separate PR?

Reply:
I'm not sure about that, I think we should discuss it before. Either way I'd create a separate PR to keep them organized.